lib/transformer.js

```diff
@@ -131,7 +131,7 @@ function createClassPrefix(classPrefix) {
 }
 
 function createIdPrefix(idPrefix) {
-    var url_pattern = /^url\(#.+\)$/i;
+    var url_pattern = /url\(#.+?\)/i;
```

Comment:
why add the question mark here ?

Reply:
to enable lazy mode.

In greedy mode, the following
`<g style="mask: url(#foo); color: rgba(0, 0, 0, 0, .1)">` 
would match 
`url(#foo); color: rgba(0, 0, 0, 0, .1)`
 but we only want
`url(#foo)`